src/microbe_stage/Spawners.cs

```diff
@@ -115,12 +115,18 @@ public static void GiveFullyGrownChanceForMulticellular(Microbe microbe, Random
             for (int i = 0; i < random.Next(Constants.MIN_BACTERIAL_COLONY_SIZE,
                      Constants.MAX_BACTERIAL_COLONY_SIZE + 1); i++)
             {
-                // Dont spawn them on top of each other because it
-                // causes them to bounce around and lag
-                yield return SpawnMicrobe(species, location + curSpawn, worldRoot, microbeScene, true,
-                    cloudSystem, currentGame);
+                // Skip spawning this microbe if it would spawn outside of the despawn radius
+                float distanceSquared = (playerPosition - (location + locationOffset)).LengthSquared();
+                float despawnRadius = Constants.MICROBE_SPAWN_RADIUS + Constants.DESPAWN_RADIUS_OFFSET;
+                if (distanceSquared < Mathf.Pow(despawnRadius, 2.0f))
```

Comment:
I'd make `Mathf.Pow(despawnRadius, 2.0f)` into a new constant in Constants to make sure it isn't calculated during runtime.

Reply:
If the value is used a lot in a squared manner, yes I'd like to see that be a constant.

Reply:
ok